main.js

```diff
@@ -22,15 +22,16 @@ function createWindow(on_load) {
     newWindow.loadURL('file://' + __dirname + '/index.html');
 }
 
-function newDocument() {
+function newProject() {
     createWindow(function(win) {
-        win.webContents.send('new-document');
+        win.webContents.send('new-project');
     });
 }
 
-function openDocument(path) {
+function openProject(path) {
     createWindow(function(win) {
-        win.webContents.send('open-document', path);
+        path = path.replace('app.tbinfo','');
```

Comment:
Actually I'm thinking the logic should live here since it applies if a document is opened via other means too (double-click from filesystem, recents menu).